### ISSUE TITLE
Descriptions for OTUs vs. Specimens

### ISSUE THREAD

**Issue by ecurrano** [paleobot/pbot_client#11]
Follow-up from today's conversation about whether descriptions are tied to OTUs or specimens. 

In most cases, descriptions will be tied to OTUs, and then one or a few specimens are linked as "type specimens" or "exemplars" for the OTU. The exception is if we are going to be doing quantitative traits (e.g. leaf mass per area, stomatal index, insect herbivory)- these refer to individual specimens and could vary considerably across an OTU. So, if we are going to be a repository for quantitative trait data, then we do need the capacity to tie descriptions to characters.

**Comment by aazaff** [comment]
Yeah, I think having the capacity to describe specimens is important to keep because it will give us some flexibility. The trick will be making it so that it is not necessary to describe a specimen in order to add it.

**Comment by aazaff** [comment]
This is subsumed into issue #12.
